src/examples/blinkm/blinkm.go

```diff
@@ -0,0 +1,38 @@
+// Connects to an BlinkM I2C RGB LED.
```

Comment:
Suggestion: add a link to the specification for the BlinkM I2C protocol.

Reply:
Done.